docs/advanced-configuration.md

```diff
@@ -515,6 +515,38 @@ autodetect:
   prometheus: true
 ```
 
+## Using feature gates
+Enable or disable features of the otel-collector agent, clusterReceiver, and gateway (respectively) using feature
+gates. Use the agent.featureGates, clusterReceiver.featureGates, and gateway.featureGates configs to enable or disable
+features, these configs will be used to populate the otelcol binary startup argument "--feature-gates". For more
+details see the
+[feature gate documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/main/service/featuregate/README.md).
+
+Helm Install Example:
+```bash
+# Due to syntax limitations, when using "," characters between feature gate identifiers you must escape them with "\\".
+helm install {name} --set agent.featureGates=+feature1 --set=clusterReceiver.featureGates=feature1\\,-feature2 {other_flags}
```

Comment:
Does `--set="clusterReceiver.featureGates=feature1,-feature2"` work instead of escaping?

Reply:
If we quote the whole value we still have to use a "\" to escape any "," (it's a helm syntax compilation thing). I'm seeing what else we can do here. We might be able to use a helper function here that accepts strings or lists (which might be cleaner).

Reply:
Ok, it's very unlikely that users need more than one feature gate at a time. Let's simplify it to one feature gate per each --set. User can always define them in yaml if needed

Reply:
Done